google-cloud-storage/src/test/java/org/jclouds/googlecloudstorage/features/ObjectApiLiveTest.java

```diff
@@ -422,8 +425,9 @@ public void testDeleteObjectWithOptions() {
 
    @AfterClass
    private void deleteBucket() {
-      api.getBucketApi().deleteBucket(BUCKET_NAME);
-      api.getBucketApi().deleteBucket(BUCKET_NAME2);
+      assertTrue(api.getBucketApi().deleteBucket(BUCKET_NAME));
+      assertTrue(api.getBucketApi().deleteBucket(BUCKET_NAME2));
+      assertFalse(api.getBucketApi().deleteBucket(NONEXISTENT_BUCKET_NAME));
```

Comment:
Should `@AfterClass` have asserts?  This method should just clean up after the test mehod.  Instead can you create a separate `testDeleteBucket` method?